simple-git-hooks.test.js

```diff
@@ -275,7 +291,60 @@ test.each([
 
     spc.setHooksFromConfig(projectWithCustomConfigurationFilePath, args)
     const installedHooks = getInstalledGitHooks(path.normalize(path.join(projectWithCustomConfigurationFilePath, '.git', 'hooks')))
-    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({'pre-commit':`#!/bin/sh\nexit 1`, 'pre-push':`#!/bin/sh\nexit 1`}))
-
-    removeGitHooksFolder(projectWithCustomConfigurationFilePath)
+    expect(JSON.stringify(installedHooks)).toBe(JSON.stringify({'pre-commit':`${spc.PREPEND_SCRIPT}exit 1`, 'pre-push':`${spc.PREPEND_SCRIPT}exit 1`}))
 })
+
+
+test("bypasses hooks when SKIP_SIMPLE_GIT_HOOKS is set to 1", () => {
+  execSync("git init \
+    && git config user.name github-actions \
+    && git config user.email github-actions@github.com", { cwd: projectWithConfigurationInPackageJsonPath });
+
+   createGitHooksFolder(projectWithConfigurationInPackageJsonPath);
+
+   spc.setHooksFromConfig(projectWithConfigurationInPackageJsonPath);
+   process.env.SKIP_SIMPLE_GIT_HOOKS = '1';  // Set environment variable
```

Comment:
I think you might need to unset variable at the end of test case.. Maybe move to after Each ? Not sure